jsignature/utils.py

```diff
@@ -29,24 +29,36 @@ def _remove_empty_pts(pt):
         raise ValueError
 
     # Compute box
-    width = int(round(max(chain(*[d['x'] for d in drawing])))) + 10
-    height = int(round(max(chain(*[d['y'] for d in drawing])))) + 10
+    min_width = int(round(min(chain(*[d['x'] for d in drawing])))) - 10
+    max_width = int(round(max(chain(*[d['x'] for d in drawing])))) + 10
+    width = max_width - min_width
+    min_height = int(round(min(chain(*[d['y'] for d in drawing])))) - 10
+    max_height = int(round(max(chain(*[d['y'] for d in drawing])))) + 10
+    height = max_height - min_height
 
     # Draw image
     im = Image.new("RGBA", (width * AA, height * AA))
     draw = ImageDraw.Draw(im)
     for line in drawing:
         len_line = len(line['x'])
-        points = [(line['x'][i] * AA, line['y'][i] * AA)
-                  for i in range(0, len_line)]
+        points = [
+            (
+                (line['x'][i] - min_width) * AA,
+                (line['y'][i] - min_height) * AA
+            )
+            for i in range(0, len_line)
+        ]
         draw.line(points, fill="#000", width=2 * AA)
     im = ImageOps.expand(im)
     # Smart crop
     bbox = im.getbbox()
     if bbox:
         im.crop(bbox)
 
-    im.thumbnail((width, height), Image.ANTIALIAS)
+    im.thumbnail(
+        (width, height),
+        Image.ANTIALIAS if PIL_VERSION < '10.0.0' else Image.LANCZOS
+    )
```

Comment:
This won't work as for example 
```
>>> '9.0.0' < '10.0.0'
False
```

Instead you can do

```suggestion
    pil_version_tuple = tuple(map(int, PIL_VERSION.split('.'))) 
    im.thumbnail(
        (width, height),
        Image.ANTIALIAS if pil_version_tuple < (10, 0, 0) else Image.LANCZOS
    )
```